api/repositories/service_instance_repository_test.go

```diff
@@ -504,6 +504,7 @@ func initializeServiceInstanceCreateMessage(serviceInstanceName string, spaceGUI
 	}
 }
 
+// TODO: refactor to use createServiceInstanceCR
```

Comment:
Is there a a chore about this TODO? If yes, having it here does not make sense, if no, please create one.

Reply:
Good call, we just did the refactor in place since the changes were only in one file.